### PR TITLE
Task 1468 download data modal

### PR DESCRIPTION
**Pull request by maciekz1996**
Changes:

* After clicking Download button in Toolbar modal informing that the full data set is going to be downloaded is shown
* Created new api endpoint `/api/cases/getDownloadUrl` responsible for generating signed url needed to download file from S3 Bucket

Screenshots:

Screen Shot 2021-01-27 at 15 01 24

### CODE REVIEW

**Review by sergioloporto — CHANGES_REQUESTED** [review]
It looks like that according tot the [issue](https://github.com/globaldothealth/list/issues/1468), the button name should be "download full datase".
And probably "dataset" instead than "data set". Like this:

image

Need to confirm with @calremmel the correct working: dataset or data set.

File: verification/curator-service/api/src/index.ts [hunk 1]
```diff
@@ -130,6 +131,7 @@ const awsEventsClient = new AwsEventsClient(
     awsLambdaClient,
     env.SERVICE_ENV,
 );
+const s3Client = new AWS.S3({ region: 'us-east-1', signatureVersion: 'v4' });
```
Comment (sergioloporto) [inline verification/curator-service/api/src/index.ts:134]: Is it right, or should it be us-east-2?

**Review by sergioloporto — CHANGES_REQUESTED** [review]

File: verification/curator-service/api/src/index.ts (hunk 1 above)
Comment (calremmel) [reply verification/curator-service/api/src/index.ts:134]: us-east-1 is correct! many of the s3 buckets are on 1

**Review by sergioloporto — APPROVED** [review]

**Comment by calremmel** [comment]
@maciekz1996 This looks great. I noticed that the Cypress tests failed for Chrome - looks like for some reason it couldn't run them. Any thoughts for why? Let me know if it is safe to ignore.

**Comment by maciekz1996** [comment]
@calremmel The Cypress tests were all passing before changing button text from "Download" to "Download full dataset". I noticed that Cypress test window is small and because of that input in SearchBar becomes too small to be focused by Cypress `cy.type(...)` due to wider button with longer text next to it. Let me know if I should try to resolve this issue.

**Comment by maciekz1996** [comment]
@calremmel The issue with Cypress tests failing is now fixed

**Pull request merged**